src/Composer/Satis/Builder/ArchiveBuilder.php

```diff
@@ -99,9 +101,19 @@ public function dump(array $packages)
                 if (!$this->skipErrors) {
                     throw $exception;
                 }
+
+                //If file exists then exclude it
```

Comment:
@JamesRezo if `$this->skipErrors===false` then `dump()` method will stop and no files will be deleted